### PR TITLE
docs: describe how `actor_id` / `integration_id` can be found

### LINKED ISSUE #2473: [DOCS]: Document how `actor_id` / `integration_id` can be found
### Describe the need

re-submit of https://github.com/integrations/terraform-provider-github/issues/1893

I wish it's clear how the values can be found.

### SDK Version

provider 6.4.0

### API Version

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mikutas**
Resolves #2473

----

### Before the change?

* its' unclear what `integration_id` is and how to get.

### After the change?

* Users can find how to get `integration_id`.

### Pull request checklist
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [x] No

----

### CODE REVIEW

**Comment by stevehipwell** [comment]
@mikutas do you think it'd be worth adding the value for GitHub Actions directly into the docs? I think it's a global value of `15368` for everyone.

```shell
gh api \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /apps/github-actions
```

File: website/docs/r/repository_ruleset.html.markdown [hunk 1]
```diff
@@ -179,7 +179,7 @@ The `rules` block supports the following:
 
 * `context` - (Required) (String) The status check context name that must be present on the commit.
 
-* `integration_id` - (Optional) (Number) The optional integration ID that this status check must originate from.
+* `integration_id` - (Optional) (Number) The optional integration ID that this status check must originate from. It's an GitHub App ID, which can be obtained from [Get an app API](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-an-app)
```
Comment (kfcampbell) [inline website/docs/r/repository_ruleset.html.markdown:182]:
```suggestion
* `integration_id` - (Optional) (Number) The optional integration ID that this status check must originate from. It's a GitHub App ID, which can be obtained by following instructions from the [Get an App API docs](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-an-app).
```

File: website/docs/r/repository_ruleset.html.markdown [hunk 2]
```diff
@@ -205,7 +205,7 @@ The `rules` block supports the following:
 
 #### bypass_actors ####
 
-* `actor_id` - (Required) (Number) The ID of the actor that can bypass a ruleset.
+* `actor_id` - (Required) (Number) The ID of the actor that can bypass a ruleset. If `actor_type` is `Integration`, `actor_id` is an GitHub App ID. App ID can be obtained from [Get an app API](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-an-app)
```
Comment (kfcampbell) [inline website/docs/r/repository_ruleset.html.markdown:208]:
```suggestion
* `actor_id` - (Required) (Number) The ID of the actor that can bypass a ruleset. If `actor_type` is `Integration`, `actor_id` is a GitHub App ID. App ID can be obtained by following instructions from the [Get an App API docs](https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#get-an-app)
```

**Review by kfcampbell — APPROVED** [review]
This seems reasonable to me, thank you!

**Pull request merged**